EmVer/include/EmVerUtility.hpp

```diff
@@ -0,0 +1,5 @@
+#include "mbed.h"
+
+bool assertPin(PinName pinName, bool expected, int timeout);
```

Comment:
I am a huge fan of having units within the variable name. I would suggest naming timeout as timeoutMs.